### DISCUSSION TITLE
Prisma, custom serializer for date

### DISCUSSION THREAD

**Discussion by feelixe** [prisma/prisma#18179] [Q&A] [answered]
**EDIT**: Nvm, just saw this question was already answered.

Since you can pass a Date object as a value when calling .create on a model,  I would assume there's some serializer that's converting the Date object to a value that can be inserted into the database.

My question is whether it's possible to write a custom serializers for objects in Prisma. For example, maybe I don't want to use the native Date object and want to use a [dayjs](https://www.npmjs.com/package/dayjs) object instead.

I can of course manually convert the date before and after creation and retrieve but that's not a very convenient solution.

**Comment by nurul3101** [comment]
Hello @Feelixe-tin 👋 

Thanks for letting us know that the query is answered. It would be helpful if you can share the link to the discussion where the question was answered, it will help other users who might run into this.

**Reply by feelixe** [reply]
You're right, I've added a comment

**Reply by nurul3101** [reply]
Thanks for adding the link 🙌

**Comment by feelixe** [comment, marked as answer]
The question has been answered here.
#5522
